engine/swarm/admin_guide.md

```diff
@@ -53,6 +53,29 @@ updates to the swarm. A quorum of managers must also agree on node additions
 and removals. Membership operations are subject to the same constraints as state
 replication.
 
+### Maintaining a quorum of managers
+
+Always maintain **an odd number of manager nodes** in the swarm. Otherwise, your
+swarm may lose a quorum of managers, and will not be able to perform management
+tasks. Three managers is the minimum, and five managers is recommended. In a
```

Comment:
Should we combine this with https://github.com/mstanleyjones/docker.github.io/blob/0db3e0bf877880a935d53b14459ee9bd09786656/engine/swarm/admin_guide.md#add-manager-nodes-for-fault-tolerance, as it's largely repeating the same message?

Note that having a single manager is "ok" (if no fault tolerance is needed at all)